lib/list.js

```diff
@@ -238,7 +270,17 @@ function listRemoteAsync(filter) {
  * @param remoteName Required name of one of the remotes configured in settings.json.
  * @returns An array of version objects.
  */
+getRemoteVersionsAsync.cache = new Map();
```

Comment:
nitpick: I am sorry to say I think this assignment does not belong between a function and its corresponding JSDoc comments. I can only imagine what would happen if we would try running `tsc` in strict mode.

And I am now wondering why this should be a visible, undocumented member of the function in the first place. Couldn't we just define this as a `const` near the beginning, say right after `const githubReleasesRegex`?

Reply:
Good point. I had overlooked this. Surprisingly, typescript doesn't complain about it when I run in type-checking mode against the JS code. But I'll clean it up.

Reply:
Fixed in db5548e